snapshot/git/gitfiler/pool.go

```diff
@@ -40,7 +43,8 @@ func NewRepoPool(initer RepoIniter,
 // Cf. sync's Pool
 // Supports maximum pool capacity - 0 is unlimited
 type RepoPool struct {
-	initer RepoIniter
+	initer  RepoIniter
+	statRec stats.StatsReceiver
```

Comment:
Makes sense, was trying to avoid var name conflict, but not really an issue in this case